### PR TITLE
Update of organisation activity 

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Update of organisation activity + some stuff

### CODE REVIEW

File: apps/festival/festival/src/app/landing/landing.component.ts [hunk 1]
```diff
@@ -15,9 +15,11 @@ export class LandingComponent {
     distinctUntilChanged(),
     map(isTop => isTop ? 'transparent-toolbar' : '')
   );
-
+  
+  // According to this article, it's fine with Angular Universal
+  // source: https://technoapple.com/blog/post/scroll-event-at-angular-universal
```
Comment (phpgeekfr) [inline apps/festival/festival/src/app/landing/landing.component.ts:20]: this is for seo purposes?

File: libs/utils/src/lib/pipes/to-label.pipe.ts [hunk 2]
```diff
@@ -9,14 +9,17 @@ const registeredObjects = [
   'contractStatus',
   'distributionDealStatus',
   'contractType',
-];
+  'orgActivity'
+] as const;
+
+type Label = typeof registeredObjects[number];
```
Comment (phpgeekfr) [inline libs/utils/src/lib/pipes/to-label.pipe.ts:15]: nice

File: libs/organization/src/lib/organization/layout/view/view.component.html [hunk 3]
```diff
@@ -16,7 +16,7 @@
       <div fxLayout fxLayoutAlign="end" fxLayoutGap="16px">
         <div fxLayout="column" fxLayoutAlign="space-around end">
           <span>{{ org.denomination.full }}</span>
-          <span class="mat-caption">{{ org.activity }}</span>
+          <span class="mat-caption">{{ org.activity | toLabel: 'orgActivity' }}</span>
```
Comment (phpgeekfr) [inline libs/organization/src/lib/organization/layout/view/view.component.html:19]: | toLabel pipe is async. you need a | async

File: libs/utils/src/lib/dynamic-title/dynamic-title.service.ts [hunk 4]
```diff
@@ -46,6 +57,7 @@ export class DynamicTitleService implements OnDestroy {
 
   constructor(private title: Title, private routerQuery: RouterQuery) {
     this.sub = this.routerQuery.select('state').subscribe(data => {
+      console.log('data', data);
```
Comment (phpgeekfr) [inline libs/utils/src/lib/dynamic-title/dynamic-title.service.ts:60]: Forgotten to remove this console.log  ?

File: libs/organization/src/lib/organization/forms/organization-form/organization-form.component.ts [hunk 5]
```diff
@@ -11,6 +12,7 @@ import { OrganizationForm } from '@blockframes/organization/organization/forms/o
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class OrganizationFormComponent {
+  activities = activities;
```
Comment (phpgeekfr) [inline libs/organization/src/lib/organization/forms/organization-form/organization-form.component.ts:15]: no "private" or "public" means public ?

File: libs/organization/src/lib/organization/+state/organization.firestore.ts [hunk 6]
```diff
@@ -37,6 +37,26 @@ interface OrganizationRaw<D> {
...
+  filmFund: 'Film Fund',
+  filmLibrary: 'Film Library',
+  filmCommission: 'Film Commission',
+  financialInstitution: 'Financial Institution',
+  press: 'Press',
+} as const;
+
+type OrgActivity = keyof typeof activities;
+
+// Theatrical Exhibition, Distribution, Buyers’ Rep, Film Festival, Film Fund, Film Library, Film Commission, Financial Institution, Press
```
Comment (phpgeekfr) [inline libs/organization/src/lib/organization/+state/organization.firestore.ts:58]: forgotten to remove this comment ?

File: apps/festival/festival/src/app/marketplace/organization/list/list.component.html [hunk 7]
```diff
@@ -2,7 +2,7 @@
   <mat-card>
     <img mat-card-image [imgRef]="org.logo" placeholderAsset="Organization_250.png" alt="organization logo" />
     <mat-card-content class="org-info">
-      <h4>{{ org.activity }}</h4>
+      <h4>{{ org.activity | toLabel: 'orgActivity' }}</h4>
```
Comment (phpgeekfr) [inline apps/festival/festival/src/app/marketplace/organization/list/list.component.html:5]: | toLabel pipe is async. you need a | async

File: libs/organization/src/lib/organization/+state/organization.firestore.ts [hunk 8]
```diff
@@ -18,7 +18,7 @@ interface Denomination {
 /** Document model of an Organization */
 interface OrganizationRaw<D> {
   id: string;
-  activity: string;
+  activity: OrgActivity;
```
Comment (phpgeekfr) [inline libs/organization/src/lib/organization/+state/organization.firestore.ts:21]: will update draw.io accordingly

Comment (phpgeekfr) [reply libs/organization/src/lib/organization/+state/organization.firestore.ts:21]: updated

File: apps/festival/festival/src/app/landing/landing.component.ts (hunk 1 above)
Comment (GrandSchtroumpf) [reply apps/festival/festival/src/app/landing/landing.component.ts:20]: Kind of. Angular Universal helps to do prerendering (faster -> better SEO), but as it runs the code in a server we cannot depends on window object.

Comment (phpgeekfr) [reply apps/festival/festival/src/app/landing/landing.component.ts:20]: understood

File: libs/organization/src/lib/organization/forms/organization-form/organization-form.component.ts (hunk 5 above)
Comment (GrandSchtroumpf) [reply libs/organization/src/lib/organization/forms/organization-form/organization-form.component.ts:15]: yes

**Pull request merged**